cmd/create/ocmrole/cmd.go

```diff
@@ -0,0 +1,387 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ocmrole
+
+import (
+	"fmt"
+	"github.com/openshift/rosa/cmd/link/ocmrole"
+
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/spf13/cobra"
+	errors "github.com/zgalor/weberr"
+
+	"github.com/openshift/rosa/cmd/login"
+	"github.com/openshift/rosa/cmd/verify/oc"
+	"github.com/openshift/rosa/cmd/verify/quota"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/aws/tags"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var args struct {
+	prefix              string
+	permissionsBoundary string
+
+}
+
+var Cmd = &cobra.Command{
+	Use:     "ocm-role",
+	Aliases: []string{"ocmrole"},
+	Short:   "Create IAM role for OCM purposes.",
+	Long:    "Create .",
+	Example: `  # Create default ocm role for ROSA clusters using STS
+  rosa create ocm-role
+
+  # Create account roles with a specific permissions boundary
+  rosa create ocm --permissions-boundary arn:aws:iam::123456789012:policy/perm-boundary`,
+	Run: run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	flags.StringVar(
+		&args.prefix,
+		"prefix",
+		aws.DefaultPrefix,
```

Comment:
we dont want default prefix i think for this one . @vkareh can confirm though

Reply:
I think for this role the default prefix may make sense to keep it consistent... I suspect most users will have only one set of account roles, and have this match is probably fine. @arendej do you have strong opinions either way?